dashboard/src/locales/messages/index.ts

```diff
@@ -201,6 +201,8 @@ export const messages = {
     'hardwareDetails.platforms': 'Platforms',
     'hardwareDetails.timeFrame':
       'Results from {startDate} and {startTime} to {endDate} {endTime}',
+    'hardwareListing.description': 'List of hardwares from kernel tests',
```

Comment:
"hardware" is uncountable

```suggestion
    'hardwareListing.description': 'List of hardware from kernel tests',
```

or


```suggestion
    'hardwareListing.description': 'List of hardware components from kernel tests',
```